playground-app/src/scenarios/fraction-string.ts

```diff
@@ -3,9 +3,9 @@ import { contextToPrettyJson, featureDefinitionToPrettyJson } from "../utils";
 
 export const pseudoRandomSplit: Scenario = {
   description: [
-    'In this scenario, we have a feature flag with the key "pseudo-random-split" that is enabled and has four variants: red, blue, green, and grey.',
+    'In this scenario, we have a feature flag with the key "color-palette-experiment" that is enabled and has four variants: red, blue, green, and grey.',
     'The targeting rule uses the "fractional" operator, which deterministically splits the traffic based on the configuration.',
-    "This configuration splits the traffic evenly between the four variants but notice that the targeting rule is based on the sessionId, which means that the same user will always get the same variant.",
+    'This configuration splits the traffic evenly between the four variants by bucketing evaluations using the "targetingKey" and feature flag key.',
```

Comment:
```suggestion
    'This configuration splits the traffic evenly between the four variants by bucketing evaluations pseudorandomly using the "targetingKey" and feature flag key.',
```

Optional, but helpful I think. Spell check flags this, but I think it's a valid term-of-art: https://en.wiktionary.org/wiki/pseudorandomly